crates/e2e/tests/e2e/eth_integration.rs

```diff
@@ -125,6 +123,6 @@ async fn eth_integration(web3: Web3) {
     );
     assert_eq!(
         trader_b_eth_balance_after - trader_b_eth_balance_before,
-        49_776_284_258_574_379_601_u128.into()
```

Comment:
These tests strike me as useful as a chocolate teapot, basically whenever they fail we just rerun the assertion and update the value (so it's not really testing anything useful).

Reply:
I think the proper assert here would be 
trader_b_eth_balance_after >= 49 eth

Reply:
fixed as suggested by myself 😄